src/Package/Test/Browsers/WebBrowserServicesTest.cs

```diff
@@ -28,49 +25,5 @@ public class WebBrowserServicesTest {
             wbs.OpenBrowser(WebBrowserRole.External, _url);
             ps.Received().Start(_url);
         }
-
-        [Test]
-        [Category.R.Package]
-        public void WebHelpBrowser() {
-            var externalSettings = Substitute.For<IRToolsSettings>();
-            externalSettings.WebHelpSearchBrowserType.Returns(BrowserType.External);
-
-            var internalSettings = Substitute.For<IRToolsSettings>();
-            internalSettings.WebHelpSearchBrowserType.Returns(BrowserType.Internal);
-
-            RunBrowserTest(WebBrowserRole.Help, RGuidList.WebHelpWindowGuid, Resources.WebHelpWindowTitle, externalSettings, internalSettings);
-        }
-
-        [Test]
-        [Category.R.Package]
-        public void ShinyBrowser() {
-            var externalSettings = Substitute.For<IRToolsSettings>();
-            externalSettings.HtmlBrowserType.Returns(BrowserType.External);
-
-            var internalSettings = Substitute.For<IRToolsSettings>();
-            internalSettings.HtmlBrowserType.Returns(BrowserType.Internal);
-
-            RunBrowserTest(WebBrowserRole.Shiny, RGuidList.ShinyWindowGuid, Resources.ShinyWindowTitle, externalSettings, internalSettings);
-        }
-
-        public void RunBrowserTest(WebBrowserRole role, Guid guid, string title, IRToolsSettings externalSettings, IRToolsSettings internalSettings) {
-            var vswbs = Substitute.For<IVsWebBrowsingService>();
-            var ps = Substitute.For<IProcessServices>();
-
-            var wbs = new WebBrowserServices(vswbs, ps, externalSettings);
-            wbs.OpenBrowser(role, _url);
-            ps.Received().Start(_url);
-
-            ps.ClearReceivedCalls();
-            wbs = new WebBrowserServices(vswbs, ps, internalSettings);
-            wbs.OpenBrowser(role, _url);
```

Comment:
Since there is only one now, there is not much to test. However, I can keep this just in case